### PR TITLE
adds logged in options menu

### PR DESCRIPTION
**Pull request by Jbarget**
- adjusts LOGOUT text to be an options menu on Homepage
- adds same button to query page

screen shot 2018-09-19 at 10 08 48

screen shot 2018-09-19 at 10 08 58

### CODE REVIEW

File: client/src/OptionsMenu/OptionsMenu.elm [hunk 1]
```diff
@@ -88,7 +102,7 @@ optionsMenu { menuId, menuTitle, isOpen, openMsg, closeMsg, focusMsg, focusItem,
                                         item
 
                                     itemId =
-                                        menuId ++ "-options__item-" ++ toString index
+                                        menuId ++ "__item-" ++ toString index
```
[the remark this answers is not in the document]
Comment (Jbarget) [reply client/src/OptionsMenu/OptionsMenu.elm]: silly me

File: client/src/OptionsMenu/OptionsMenu.elm [hunk 2]
```diff
@@ -21,6 +21,8 @@ type alias OptionsMenuItem =
 type alias OptionsMenuModel =
     { menuId : String
     , menuTitle : String
+    , menuPositionClass : String
+    , headerTextTransform : String
```
[the remark this answers is not in the document]
Comment (Jbarget) [reply client/src/OptionsMenu/OptionsMenu.elm]: option 3

File: client/src/Update/Update.elm [hunk 3]
```diff
@@ -571,6 +572,19 @@ update msg model =
             , Dom.focus ("snippet-options__item-" ++ toString index) |> Task.attempt FocusResult
             )
 
+        LoggedInNavigationOptionsMenuOpen ->
```
[the remark this answers is not in the document]
Comment (Jbarget) [reply client/src/Update/Update.elm]: very much agree

File: client/src/css/_custom.css [hunk 4]
```diff
@@ -176,3 +185,7 @@ button {
 .loading-opacity {
   animation: loading-opacity 2s infinite;
 }
+
+.ml--1-3 {
```
[the remark this answers is not in the document]
Comment (Jbarget) [reply client/src/css/_custom.css]: me too but not sure what to do if the `Category` and `Snippet` need it but the `LoggedInNavigation` doesnt

[the remark this answers is not in the document]
Comment (mantagen) [reply client/src/css/_custom.css]: Mmmm yeah true. Don't really know.
One option is for it to have class name `category-options-menu` or `header-options-menu` etc (derived from the menuId) and then use css to style any differences, most will be the same, might just me abit of `left` `right` `top` etc

**Pull request merged**